### DISCUSSION TITLE
HomeBrew 能不能安排一下？

### DISCUSSION THREAD

**Discussion by ShinChven** [nextai-translator/nextai-translator#445] [General]
手动更新好麻烦哎，要不上 AppStore 挂个几美元吧，我去买。

**Comment by Macin20** [comment]
支持，求 brew cask 支持 ⬆️

**Reply by ShinChven** [reply]
https://github.com/ShinChven/gh-release-dl

我还写了一个这个来下载。

**Comment by ivaquero** [comment]
可以暂时使用仓库 [homebrew-extras](https://github.com/Brewforge/homebrew-extras)

```bash
brew tap brewforge/extras
brew install openai-translator
```
